index.html

```diff
@@ -0,0 +1,319 @@
+                        </li>
+                    </ul>
+                    <p class="pry-text">A daily selection of privately personalized reads; no accounts or sign-ups
+                        required.</p>
+                    <ul class="tags">
+                        <li class="tag1">
+                            <a href="#">
+                                <p class="html">html</p>
+                            </a>
+                        </li>
+                        <li class="tag2">
+                            <a href="#">
+                                <p class="css">css</p>
+                            </a>
+                        </li>
+                        <li class="tag3">
+                            <a href="#">
+                                <p class="js">javaScript</p>
+                            </a>
+                        </li>
+                    </ul>
+
+                </div>
+
+                <div class="action">
+                    <button class="see-project-l">
+                        <p class="see-p">See Project</p>
+                    </button>
+                </div>
+            </div>
+        </div>
+        <div class="card-works3">
+            <div class="snap3"></div>
+            <div class="left-block">
+                <div class="pt">
+                    <h2>Tonic</h2>
+                    <ul class="frame2">
+                        <li class="client">
+                            <h5 class="canopy">CANOPY</h5>
+                        </li>
+                        <li class="dot"></li>
+                        <li class="role">
+                            <h5 class="Back">Back End Dev</h5>
+                        </li>
+                        <li class="dot"></li>
+                        <li class="year-label">
+                            <h5 class="year">2015</h5>
+                        </li>
+                    </ul>
+                    <p class="pry-text">A daily selection of privately personalized reads; no accounts or sign-ups
+                        required.</p>
+                    <ul class="tags">
+                        <li class="tag1">
+                            <a href="#">
+                                <p class="html">html</p>
+                            </a>
+                        </li>
+                        <li class="tag2">
+                            <a href="#">
+                                <p class="css">css</p>
+                            </a>
+                        </li>
+                        <li class="tag3">
+                            <a href="#">
+                                <p class="js">javaScript</p>
+                            </a>
+                        </li>
+                    </ul>
+
+                </div>
+
+                <div class="action">
+                    <button class="see-project">
+                        <p class="see-p">See Project</p>
+                    </button>
```

Comment:
- [ ] Just as stated by the previous reviewer , please style the button appropriately. On hover, the button does not react. and when clicked, it has this color
![image](https://user-images.githubusercontent.com/50990925/219665639-00c543e2-61b2-4531-ab36-5417629c0d9f.png)
But should have this:
![image](https://user-images.githubusercontent.com/50990925/219665708-a00b3628-6c42-4c60-8d40-1499c9536499.png)

- [ ] On every hover, on all buttons, it should have this :
![image](https://user-images.githubusercontent.com/50990925/219665847-31c8b209-b718-49c6-a617-5aa3c03d8865.png)
- When the hover is shifted away, it should be interactive to return to normal. as it stands, it doesn't not. Kindly fix that 👍